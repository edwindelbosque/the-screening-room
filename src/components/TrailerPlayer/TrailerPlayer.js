import React from 'react';

export const TrailerPlayer = ({ youtubeId }) => {
  return (
    <div
      className='video'
      style={{
        position: 'relative',
        paddingBottom: '48%',
        paddingTop: 25,
        height: 0
      }}
    >
      <iframe
        title={youtubeId}
        style={{
          position: 'absolute',
          top: 0,
          left: 0,
          width: '100%',
          height: '100%'
        }}
        src={`https://www.youtube.com/embed/${youtubeId}?controls=0;&autoplay=1`}
        frameBorder='0'
        allowFullScreen
        autoPlay
      />
    </div>
  );
};
