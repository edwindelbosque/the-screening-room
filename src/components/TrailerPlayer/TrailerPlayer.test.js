import React from 'react';
import { shallow } from 'enzyme';
import { TrailerPlayer } from './TrailerPlayer';

describe('TrailerPlayer', () => {
  let wrapper;
  let youtubeId = 'string';

  beforeEach(() => {
    wrapper = shallow(<TrailerPlayer youtubeId={youtubeId} />);
  });

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
