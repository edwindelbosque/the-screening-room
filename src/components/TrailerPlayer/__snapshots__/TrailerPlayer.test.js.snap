// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TrailerPlayer should match the snapshot 1`] = `
<div
  className="video"
  style={
    Object {
      "height": 0,
      "paddingBottom": "48%",
      "paddingTop": 25,
      "position": "relative",
    }
  }
>
  <iframe
    allowFullScreen={true}
    autoPlay={true}
    frameBorder="0"
    src="https://www.youtube.com/embed/string?controls=0;&autoplay=1"
    style={
      Object {
        "height": "100%",
        "left": 0,
        "position": "absolute",
        "top": 0,
        "width": "100%",
      }
    }
    title="string"
  />
</div>
`;
